lib/PuppeteerSharp/Page.cs

```diff
@@ -1284,7 +1284,9 @@ public Task<ElementHandle> WaitForXPathAsync(string xpath, WaitForSelectorOption
             var responses = new Dictionary<string, Response>();
 
             EventHandler<ResponseCreatedEventArgs> createResponseEventListener = (object sender, ResponseCreatedEventArgs e) =>
-                responses.Add(e.Response.Url, e.Response);
+            {
+                responses[e.Response.Url] = e.Response;
+            };
```

Comment:
@Meir017 would you use brackets here?

Reply:
I will remove it to keep the code style consistent